packages/create-next-app/templates/default/pages/index.js

```diff
@@ -50,7 +50,7 @@ const Home = () => (
 
     <footer>
       <a
-        href="https://zeit.co?utm_source=create-next-app&utm_medium=referral&utm_campaign=Create%20Next%20App"
+        href="https://zeit.co?utm_source=create-next-app&utm_medium=default-example&utm_campaign=examples"
```

Comment:
```suggestion
        href="https://zeit.co?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
```
Shouldn't these still be the same campaign as above since you wouldn't really use the default template from CNA without CNA right?